sdk/digitaltwins/digitaltwins/swagger/digitaltwins.json

```diff
@@ -0,0 +1,1497 @@
+{
```

Comment:
why is the swagger being checked into this repo? I think it should live here instead: https://github.com/Azure/azure-rest-api-specs/

Reply:
I agree and eventually it will be only in azure-rest-api-spec. But until we don't have a way to auto-generate from there, I followed the example of the digitaltwins .NET SDK and checked into this repo. (They have the swagger checked in to azure SDK folder) If you strongly against that I will remove it from here. It is not a big deal.

---
In reply to: [447264662](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447264662) [](ancestors = 447264662)

Reply:
I'm not sure about `auto-generate`, but we simply reference the external swagger by URL in other client libraries.

Reply:
YEs, but if there is no auto-generate in the case of new version of swagger it is better to keep the original swagger what is the code generated from. Is'n it?

---
In reply to: [453892297](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r453892297) [](ancestors = 453892297)

Reply:
That would make sense to me, except what about generating the other language SDKs? Having the spec live in some language neutral place has the advantage of allowing all languages to reference the same spec.

For cognitive search we used that to drive changes across languages (e.g. the service team updated the swagger in one place then we regenerated all clients.)

Reply:
Removed

---
In reply to: [453986003](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r453986003) [](ancestors = 453986003)

Reply:
Removed

---
In reply to: [453977418](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r453977418) [](ancestors = 453977418,453892297)

Reply:
Removed

---
In reply to: [453758091](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r453758091) [](ancestors = 453758091,447264662)